npm-shrinkwrap.json

```diff
@@ -1605,7 +1683,7 @@
         "isarray": {
           "version": "0.0.1",
           "from": "isarray@0.0.1",
-          "resolved": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz"
+          "resolved": "http://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz"
```

Comment:
Alright, I'll fix tomorrow. Perhaps I'll add it to the "fix shrinkwrap" gulp task!